tests/test_user/test_change_user_role.py

```diff
@@ -50,3 +53,13 @@ def test_assign_role_to_none_existing_user(self):
             assign_role_to_non_existing_user_mutation,
             "User not found"
         )
+
+    def test_change_user_role_with_already_assigned_role(self):
+        """
+        Testing change of user role
```

Comment:
This description is vague. It does not tell what aspect is being tested when changing user role.